packages/create/bin/create-stacks-workspace.ts

```diff
@@ -155,12 +156,14 @@ async function getConfiguration(argv: yargs.Arguments<CreateStacksArguments>) {
 async function main(parsedArgv: yargs.Arguments<CreateStacksArguments>) {
     const { nxVersion, ...forwardArgv } = parsedArgv;
     const { name } = forwardArgv;
-    const argumentsToForward = unparse(forwardArgv as unparse.Arguments, {
-        alias: {
-            packageManager: ['pm'],
-            interactive: ['i'],
-        },
-    });
+    const argumentsToForward = normaliseForwardedArgv(
```

Comment:
The code in `normaliseForwardedArgv` seems complicated because its dealing with the unparsed args. Could we do something like 
```ts
function normaliseForwardedArgv(forwardArgv) {
  forwardArgv.preset = forwardArgv.preset === 'next' ? 'apps' : forwardArgvPreset;
  return forwardArgv;
}

... main

const argumentsToForward = unparse(normaliseForwardedArgv(forwardArgv), { alias: { packageManager: ['pm'], interactive: ['i']);



```

Reply:
Worth noting that whatever we do here, we need to keep reference to the original preset for the methods like `getStacksPlugins` below, which do different things if we're setting up Next

Reply:
Think thats fine, because we pass the original parsedArgv to the methods below. Can easily have normalise clone the forwardArgv too